pom.xml

```diff
@@ -82,131 +82,61 @@
   </properties>
 
   <build>
-    <plugins>
```

Comment:
Moved the explicit plugin definition to `pluginsManagement` so that they can be pulled in by the respective parent POMs (only if needed).